recipes/rpclib/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup()
set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

that seems to be a reason why it doesn't create an import module - because no symbols are exported
thus, hook complains it cannot find a library

@uilianries do you think we can improve an error message to be less obscure, and suggesting this variable?

---

Thank you!  Updated!